setup.py

```diff
@@ -51,5 +48,9 @@ def read(*rnames):
       ],
       packages=['gcovr'],
       keywords=['utility'],
-      scripts=scripts
+      entry_points={
+          'console_scripts': [
+              'gcovr=gcovr.__main__:main',
```

Comment:
Finally 🎉 :) By moving the whole script to `__main__`, you've managed to introduce entry_points much earlier than I would have expected!

Reply:
`pip install` on Windows automatically creates `.exe` wrapper in Scripts directory due to this change! It's great! No more manual `.bat` wrappers necessary for cross-platform `gcovr` calls! :+1: